src/browser/api/window.js

```diff
@@ -558,12 +541,12 @@ Window.create = function(id, opts) {
                     windowTeardown();
                     // These were causing an exception on close if the window was reloaded
                     _boundsChangedHandler.teardown();
-                    teardownListeners();
+                    browserWindow.removeAllListeners();
```

Comment:
Remove all listeners at once.
At this point its WebContents is already gone, BrowserWindow still alive with all of its listeners, and this is the last event. Nothing should be attached to it after this...